modules/terraform.nix

```diff
@@ -928,7 +943,18 @@ let
   localExecType = with lib.types;
     submodule {
       options = {
-        protoCommand = lib.mkOption { type = with lib.types; functionTo str; };
+        protoCommand = lib.mkOption {
+          type = with lib.types; functionTo str;
+          description = "Provisioner command to be applied to all nodes";
+        };
+
+        bootstrapperCommand = lib.mkOption {
+          type = with lib.types; nullOr (functionTo str);
```

Comment:
we have `with lib.types` above already. I'd prefer `inherit` instead, since it's faster and easier to understand (and gives better error messages).